main.js

```diff
@@ -0,0 +1,74 @@
+// Tracks Board State //
+let gameBoardObj = {
+  1: [5, 4, 3, 2, 1],
+  2: [],
+  3: []
+}
+
+
+
+
+// Handles Opening Display of Game Board //
+let renderGame = function() {Object.values(gameBoardObj).map((game) => {
+  const pegs = game.toString()
+  let gameString = `--- ${pegs}`
+  
+  let gameDisplay = gameString.split(",").join(" ")
+  
+  
+  console.log(gameDisplay)
+  
+})}
+
+// Renders Game Board in Console //
+renderGame()
+
+// Checks For Win Condition and Resets the Game //
+let winState = function() {
```

Comment:
winState is not clear, just call the function for what its actually doing like checkWinner or checkWinCondition
```suggestion
let checkWinner = function() {
```